03/app.js

```diff
@@ -0,0 +1,46 @@
+document.addEventListener('DOMContentLoaded', function () {
+
+    const counterEl = document.querySelectorAll('button'); // All - wszystkie button'y
+
+    counterEl.forEach(function (button) { // zgodnie z prezentacją lepiej nie uzywać f(x) strzałkowej - tutaj nie mmay this więc w sumie można by było zostawić strzałkową 
+        let value = 0;
+
+        const incrementCounter = function () {
+            value++; // kolejne wartości
```

Comment:
Czy faktycznie musimy je zliczać? Pierwsze kliknięcie to pierwsze kliknięcie i od razu usuwamy "nasłuchiwanie". Nie trzeba nic liczyć :)